satori/resources/resource_dataset.go

```diff
@@ -141,6 +148,7 @@ func resourceDataSetRead(ctx context.Context, d *schema.ResourceData, m interfac
 
 	resultAccessControl := api.AccessControl{}
 	resultAccessControl.AccessControlEnabled = result.PermissionsEnabled
+	resultAccessControl.CustomAccessRequestsEnabled = result.CustomAccessRequestsEnabled
```

Comment:
Same here, it depends if value is returned, if it does not exist, you should not take it